cogs/admin.py

```diff
@@ -317,11 +350,14 @@ async def rulesninfo(self, ctx):
         channel = ctx.guild.get_channel(716717317320605736)
         server_rules = await channel.fetch_message(723643619315023873)
         bot_rules = await channel.fetch_message(723643619700899983)
+        #server_roles = await channel.fetch_message(ID) <- uncomment with message id
         links = await channel.fetch_message(723643620313268291)
         faqs = await channel.fetch_message(723643620946870272)
 
         await server_rules.edit(embed = server_rules_embed)
         await bot_rules.edit(embed = bot_rules_embed)
+        await channel.send(embed = server_roles_embed)  # delete this
+        #await server_roles.edit(embed = server_roles_embed) <- uncomment
```

Comment:
Do it yourself

Reply:
i can't run the command for the first time and then edit code without another pr